nuxeo-web-ui-ftest/scripts/specs-processor.js

```diff
@@ -1,9 +1,18 @@
+ const Finder = require('fs-finder');
+ const fs = require('fs');
```

Comment:
nitpick, there is an extra space before `const` keywords

Reply:
:+1: